src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -0,0 +1,122 @@
+import collections
+import paramiko
+from commcare_cloud.commands.command_base import CommandBase, Argument
+from commcare_cloud.commands.inventory_lookup.getinventory import get_instance_group
+from commcare_cloud.environment.main import get_environment
+
+
+class PrivilegedCommands():
```

Comment:
I assume you're doing this instead of running it with ansible so that you can get the command output correct?

I went through a similar issue before and settled on using Fabric to run the command which has a higher level api than paramico: https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/commands/migrations/copy_files.py#L197

You could adapt the code there to get the output as follows:
```
out = StringIO()
sudo(command, stdout=out)
```